app/src/sharedTest/java/org/oppia/android/app/profile/ProfileChooserFragmentTest.kt

```diff
@@ -296,28 +296,28 @@ class ProfileChooserFragmentTest {
     profileTestHelper.initializeProfiles()
     launch(ProfileChooserActivity::class.java).use {
       testCoroutineDispatchers.runCurrent()
-      onView(atPosition(R.id.profile_recycler_view, 0)).perform(click())
+      onView(atPosition(recyclerViewId = R.id.profile_recycler_view, position = 0)).perform(click())
```

Comment:
```suggestion
      onView(atPosition(
        recyclerViewId = R.id.profile_recycler_view,
        position = 0
      )).perform(click())
```

Similarly for `testProfileChooserFragment_clickAddProfile_opensAdminAuthActivity`

Reply:
I updated these changes